src/Tizen.NUI.Scene3D/src/public/Controls/SceneView.cs

```diff
@@ -693,5 +849,25 @@ private void OnCameraTransitionFinished(IntPtr data)
         {
             cameraTransitionFinishedEventHandler?.Invoke(this, EventArgs.Empty);
         }
+
+        // Callback for capture finished signal
+        private void OnCaptureFinished(IntPtr data, int captureId, IntPtr capturedImageUrl)
+        {
+            CaptureFinishedEventArgs e = new CaptureFinishedEventArgs();
+            ImageUrl imageUrl = new ImageUrl(NUI.Interop.ImageUrl.NewImageUrl(new ImageUrl(capturedImageUrl, false).SwigCPtr), true);
+            NDalicPINVOKE.ThrowExceptionIfExists();
+
+            e.CaptureId = captureId;
+            e.CapturedImageUrl = imageUrl;
```

Comment:
이게 아마 BaseHandle에서 `==`/`!=` 연산자에 대한 overloading이 되어 있어서 큰 문제는 없지만 (사용할 때, `== null`로 검사가 가능), 그렇지만 그럼에도 불구하고 null검사를 하지 않고 그냥 메소드를 호출하는 경우 `NullReferenceException`이 아닌 Dali가 보내는 body가 없다는 exception을 받게 되어서 이상한 것 같습니다. 

이런식으로 고치면 어떨까요?
```suggestion
            e.CapturedImageUrl = imageUrl.HasBody() ? imageUrl : null;
```

Reply:
좋은 방법인 거 같습니다.